about.html

```diff
@@ -12,53 +12,122 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="headernav">
+
+    <img class ="firstimg" src ="img/lambda-black.png">
+
+    <a class = "home" href ="index.html">Home</a>
+    <a class = "about" href ="about.html">About</a>
+    <a class = "products" href ="#">Products</a>
+    <a class = "blog" href ="#">Blog</a>
+    <a class ="contact" href ="#">Contact</a>
+
+</div>
+
+<div class="flex1">
+
+  <img class = "longpicture" src = "img/jumbo-about.png">
+
+</div>
+
+  <div class="aboutpara">
+    <h2 class ='about'>About</h2>
 
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
```

Comment:
The about page content is a good place to use `<section> and/or <article>` elements.